airbyte-cdk/python/airbyte_cdk/sources/streams/http/error_handlers/response_models.py

```diff
@@ -22,10 +26,27 @@ class ErrorResolution:
     error_message: Optional[str] = None
 
 
-DEFAULT_ERROR_RESOLUTION = ErrorResolution(
```

Comment:
This would technically be a breaking change but I don't see any usage outside of the CDK in our repo so I would keep it as a patch bump